### ISSUE TITLE
Blackberry Work (old Good)

### ISSUE THREAD

**Issue by asymcon** [devadvance/rootcloak#176]
Rootcloak2 doesn't seem to be compatible with Blackberry Work - any chance this will be implemented in the future?
Thanks

**Comment by hikaritenchi** [comment]
I think there a a lot of better options available at this point. RootCloak's biggest weakness is not being able to deal with SafetyNet, which is becoming more widely implemented at this point.

**Comment by devadvance** [comment]
Agreed. It's no longer about root, but rather system modifications in
general.
